docs/guides/embed.server.mdx

```diff
@@ -0,0 +1,115 @@
+export const navSortSelf = 5
+
+# Embed
+
+This guide explores how to embed things like tweets, gists or codepens in
+markdown.
+There are two ways to accomplish embeds: at compile time or at
+runtime.
+Doing it at compile time means a bit of extra effort is spent when compiling but
+that readers will have a very fast experience.
+Doing it at runtime gives some flexibility but depends on what framework you use
+(so it’s specific to react, preact, vue, etc.) and moves all the work to your
+users.
```

Comment:
it may also be worth noting, during compile time will produce static JSX/HTML, while runtime will include the component's JS in the runtime

Reply:
Reworded it!